backend/src/bin/scripts/process-webhook.ts

```diff
@@ -57,31 +68,88 @@ const sections = [
 const usage = commandLineUsage(sections)
 const parameters = commandLineArgs(options)
 
-if (parameters.help || !parameters.webhook) {
+if (parameters.help || (!parameters.webhook && !parameters.processErrors)) {
   console.log(usage)
 } else {
   setImmediate(async () => {
-    const webhookIds = parameters.webhook.split(',')
     const options = await SequelizeRepository.getDefaultIRepositoryOptions()
     const repo = new IncomingWebhookRepository(options)
+    let currentPage = 1
+    const PROCESS_QUEUE_THRESHOLD = 2
+    const PAGE_SIZE = 10000
```

Comment:
I would issue 10k sqs messages at once but I wouldn't load 10k webhooks into the RAM at once just in case -> for easier fix perhaps just change 10k to 1k for now.

Reply:
alright :+1:  i'll also update queue_threshold to 100